src/privatesend-client.cpp

```diff
@@ -40,13 +44,18 @@ void CPrivateSendClient::ProcessMessage(CNode* pfrom, const std::string& strComm
         CDarksendQueue dsq;
         vRecv >> dsq;
 
+        {
```

Comment:
That's for `cs_vecqueue` lock. Same logic for changes as in #2200 basically but since the block here is pretty small and there are a lot of changes anyway I guess I should probably add some whitespaces here. Will fix.